mne/io/edf/edf.py

```diff
@@ -163,6 +163,10 @@ def __init__(self, input_fname, montage, eog=None, misc=None,
             warn("Stimulus channel will not be annotated. Both 'annot' and "
                  "'annotmap' must be specified.")
 
+        if bool(annot) or bool(annotmap):
```

Comment:
doing `bool(...)` should have no effect here, it's implicitly done by the logical operations